Fix docker flaky build issue

The build inconsistently fails with the following trace: 

```
 ---> Running in 1f15227b1895
Removing intermediate container 1f15227b1895
 ---> 4650e2be875d
Step 7/8 : LABEL io.airbyte.version=0.1.0
 ---> Running in 9caa1288bd70
Removing intermediate container 9caa1288bd70
 ---> 1511c2758665
Step 8/8 : LABEL io.airbyte.name=airbyte/base-python-test
 ---> Running in b5df991cf486

Removing intermediate container b5df991cf486
FAILURE: Build failed with an exception.
 ---> 4f80736d3cb6

Successfully built 4f80736d3cb6
* What went wrong:
Successfully tagged airbyte/base-python-test:dev
Execution failed for task ':airbyte-integrations:connectors:source-facebook-marketing-api-singer:buildTestImage'.
Created image with ID '4f80736d3cb6'.
> Could not build image: pull access denied for airbyte/base-python-test, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

--- a/airbyte-integrations/connectors/source-facebook-marketing-api-singer/build.gradle
+++ b/airbyte-integrations/connectors/source-facebook-marketing-api-singer/build.gradle
@@ -29,6 +29,7 @@ build.dependsOn ':airbyte-integrations:bases:base-python-test:build'
 
 buildImage.dependsOn ':airbyte-integrations:bases:base-singer:buildImage'
 buildTestImage.dependsOn ':airbyte-integrations:bases:base-singer:buildImage'
+buildTestImage.dependsOn ':airbyte-integrations:bases:base-python-test:buildImage'
 
 integrationTest.dependsOn(buildImage)
 standardSourceTestPython.dependsOn(buildTestImage)

--- a/airbyte-integrations/connectors/source-facebook-marketing-api-singer/build.gradle
+++ b/airbyte-integrations/connectors/source-facebook-marketing-api-singer/build.gradle
@@ -28,8 +28,8 @@ build.dependsOn(unitTest)
 build.dependsOn ':airbyte-integrations:bases:base-python-test:build'
 
 buildImage.dependsOn ':airbyte-integrations:bases:base-singer:buildImage'
+integrationTest.dependsOn(buildImage)
+
 buildTestImage.dependsOn ':airbyte-integrations:bases:base-singer:buildImage'
 buildTestImage.dependsOn ':airbyte-integrations:bases:base-python-test:buildImage'
-
-integrationTest.dependsOn(buildImage)
 standardSourceTestPython.dependsOn(buildTestImage)

